src/Umbraco.Web/Editors/MediaController.cs

```diff
@@ -945,70 +945,73 @@ internal static bool CheckPermissions(IDictionary<string, object> storage, IUser
             return hasPathAccess;
         }
 
-        /// <summary>
-        /// Returns the references (usages) for the media item
-        /// </summary>
-        /// <param name="id"></param>
-        /// <returns></returns>
-        public MediaReferences GetReferences(int id)
+        public PagedResult<EntityTypeReferences> GetPagedContentReferences(int id, int pageNumber = 1, int pageSize = 100)
```

Comment:
Seems like a lot of repitition here for 3 methods that all essentially do the same thing apart from a couple of references like the type of entity and the Udi, seems like this could all exist under one method with an object type parameter and another generic method to create the result?